### PR TITLE
[ONNX] Batch_normalization operator

### PR DESCRIPTION
**Pull request by Deliadd**

### CODE REVIEW

File: pyzoo/zoo/pipeline/api/onnx/mapper/batch_norm.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#
+# Copyright 2018 Analytics Zoo Authors.
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/onnx/mapper/batch_norm.py]: batchnorm.py?

File: pyzoo/zoo/pipeline/api/onnx/mapper/batch_norm.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        return [self._to_zoo_input(self._input_list[0])]
+
+    def _to_tensor(self):
+        input = self.model_inputs[0]
+        W_weights = self.model_trainable_values[0]
+        rank = len(input.zvalue.shape)
+
+        if (rank == 4):
+
+            epsilon = 0.001
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/onnx/mapper/batch_norm.py]: these parameters should be taken from `self.onnx_attr`

**Comment by zhichao-li** [comment]
any update?

**Comment by Deliadd** [comment]
Update the current code but still with Attribute_Error in layers.normalization. 
Some part of the original code needed to be changed because no attribute 'value' is found which should be  the input of callBigDLFunc.

File: pyzoo/zoo/pipeline/api/onnx/mapper/batchnormalization.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+# limitations under the License.
+#
+from zoo.pipeline.api.onnx.mapper.operator_mapper import OperatorMapper
+from zoo.pipeline.api.onnx.onnx_helper import OnnxHelper
+import zoo.pipeline.api.keras.layers as zlayers
+import numpy as np
+
+
+class BatchNormalizationMapper(OperatorMapper):
+    bigdl_type = "float"
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/onnx/mapper/batchnormalization.py]: remove this?

**Comment by Deliadd** [comment]
Successful in Jenkins, ID is 1165

File: pyzoo/zoo/pipeline/api/onnx/onnx_loader.py [hunk 4]
```diff
@@ -56,6 +56,7 @@ def run_node(node, inputs):
 
         model = zmodels.Model(input=[i.zvalue for i in mapper.model_inputs], output=out_tensor)
         data = [i.data for i in mapper.model_inputs]
+        model.training = is_training
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/onnx/onnx_loader.py]: `model.training(is_training)` ?

File: pyzoo/zoo/pipeline/api/onnx/mapper/batchnormalization.py [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        """
+        return [self._to_zoo_input(self._input_list[0])]
+
+    def _to_tensor(self):
+        input = self.model_inputs[0]
+        rank = len(input.zvalue.shape)
+
+        if (rank == 4):
+            epsilon = float(self.onnx_attr['epsilon']) if "epsilon" in self.onnx_attr else 0.001
+            mode = self.onnx_attr['mode'] if "mode" in self.onnx_attr else 0
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/onnx/mapper/batchnormalization.py]: remove?

File: pyzoo/test/zoo/pipeline/onnx/test_model_loading.py [hunk 6]
```diff
@@ -42,6 +42,22 @@ def test_onnx_conv2d_2(self):
...
+        dim_ones = (1,) * (dims_x - 2)
+        s = s.reshape(-1, *dim_ones)
+        bias = bias.reshape(-1, *dim_ones)
+        mean = mean.reshape(-1, *dim_ones)
+        var = var.reshape(-1, *dim_ones)
+        return s * (x - mean) / np.sqrt(var + epsilon) + bias
+
+    def test_onnx_batch_norm(self):
+        pytorch_model = torch.nn.Sequential(
+            torch.nn.BatchNorm2d(num_features=3, momentum=1, affine=False)
```
Comment (zhichao-li) [inline pyzoo/test/zoo/pipeline/onnx/test_model_loading.py]: add a comment here that the `momentum` generated by pytorch is always 1.

**Comment by Deliadd** [comment]
Update code with run node unittest. Jenkins passed. ID is 1172

**Comment by Deliadd** [comment]
Update with bias and scale set

**Comment by zhichao-li** [comment]
jenkins?

**Comment by Deliadd** [comment]
Jenkins passed.  ID is 1183
